text.txt

```diff
@@ -0,0 +1 @@
+sensitive info
```

Comment:
Notice how this is still showing up in your "tracked" history, you were supposed to filter and erase/replace this text so people/bots can't search through your history. Not sure what commands you used, but I'm not supposed to see this if you used the full `filter-branch` + `git rm` command.  

-5%